modules/sandboxed-containers-deleting-catalog-subscription.adoc

```diff
@@ -0,0 +1,22 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-catalog-subscription_{context}"]
+= Deleting the {sandboxed-containers-first} catalog subscription
+
+You can delete the {sandboxed-containers-first} catalog subscription to remove namespace access to the {sandboxed-containers-operator}.
+
+.Prerequisites
+
+* You have {product-title} 1.0.0 installed.
```

Comment:
@zanetworker, @c3d, @jensfr, @cpmeadors  - thoughts?

Reply:
1.0.0 is the operator version.

Reply:
Sorry this is my fault, it should be {sandboxed-containers-operator}. However, @zanetworker I think the question is about whether or not we should be stating which version of the operator we want in a prerequisite.

Reply:
I would really go with the flow here and use the same kind of wording as for other operators.

There is no hard dependency to any version that I know of in the description given, and you can't get a lower version either since this is the first one. So if you want to omit the version, it's OK with me.

Reply:
I don't think we should, also we have not specified the operator version before in other parts of the docs.

Reply:
I will drop this prerequisite from all sections. 

* You have {product-title} 1.0.0 installed.

Reply:
I left OpenShift 4.8 as `{product-title} {product-version}`. 
I removed `{sandboxed-containers-operator} 1.0.0`